test/controllers/works_controller_test.rb

```diff
@@ -0,0 +1,153 @@
+require "test_helper"
+
+describe WorksController do
+  describe "index" do
+    it "will return success status" do
+      get works_path
+      must_respond_with :success
+    end # it "will return success status" do
+  end # index
+
+  describe "root" do
+    # check that the index page works
+    it "returns success status" do
```

Comment:
For both index and root, I would be interested in what happens if there are no works, if there are works but no votes, and if one of the categories has no works.